pkg/scheduler/framework/runtime/framework.go

```diff
@@ -678,14 +694,15 @@ func (f *frameworkImpl) RunPreFilterExtensionRemovePod(
 	podInfoToRemove *framework.PodInfo,
 	nodeInfo *framework.NodeInfo,
 ) (status *framework.Status) {
+	logger := klog.FromContext(ctx)
 	for _, pl := range f.preFilterPlugins {
 		if pl.PreFilterExtensions() == nil {
 			continue
 		}
 		status = f.runPreFilterExtensionRemovePod(ctx, pl, state, podToSchedule, podInfoToRemove, nodeInfo)
 		if !status.IsSuccess() {
 			err := status.AsError()
-			klog.ErrorS(err, "Failed running RemovePod on PreFilter plugin", "plugin", pl.Name(), "pod", klog.KObj(podToSchedule))
```

Comment:
Are really plugin, pod and possibly `RemovePod` and `PreFilter` strings already in the logger context? The same question for similar changes in other files below.

Reply:
https://github.com/kubernetes/kubernetes/pull/111155#discussion_r937049669